acmwebsite/templates/archives.xhtml

```diff
@@ -5,7 +5,7 @@
   <py:extends href="master.xhtml" />
 
   <head py:block="head" py:strip="True">
-    <title>Mines ACM - Mailing List Archives</title>
+    <title>${tg.config.get('short_site_name', 'not configured')} - Mailing List Archives</title>
```

Comment:
Default value should be in ``config/app_config.py`` and have a default of "Mozzarella".